src/InvertedIndices.jl

```diff
@@ -27,6 +27,16 @@ inverted index will similarly span multiple dimensions.
 """
 InvertedIndex, Not
 
+# A wrapper for `InvertedIndex` if it is used in broadcasting context.
+# Since `InvertedIndex` does not have a reference to the collection it applies to
+# it is impossible to define its axes properly.
+# Therefore it is the responsiblity of the caller to properly resolve the handling
+# of the broadcast result.
```

Comment:
```suggestion
"""
    BroadcastedInvertedIndex(x::Not)

A wrapper for `InvertedIndex` if it is used in broadcasting context.
Since `InvertedIndex` does not have a reference to the collection it applies to
it is impossible to define its `axes` eagerly.
Therefore it is the responsiblity of the caller to resolve the handling
of the broadcast result.

# Examples
```
julia> using InvertedIndex

julia> Not(5) .=> sum
BroadcastedInvertedIndex(InvertedIndex{Int64}(5)) => sum

julia> Not(:) .=> [minimum, maximum]
2-element Vector{Pair{BroadcastedInvertedIndex, _A} where _A}:
 BroadcastedInvertedIndex(InvertedIndex{Colon}(Colon())) => minimum
 BroadcastedInvertedIndex(InvertedIndex{Colon}(Colon())) => maximum
"""
```